ui/dashboardApp/index.ts

```diff
@@ -156,9 +156,9 @@ async function webPageStart() {
   // NOTE: Don't remove above comment line, it is a placeholder for code generator
 
   try {
-    await reloadWhoAmI()
+    const ok = await reloadWhoAmI()
```

Comment:
The spinner has already been removed in L65, will user still see a flash screen during these two requests?